cms/forms/validators.py

```diff
@@ -22,6 +24,20 @@ def validate_url(value):
         URLValidator()(value)
 
 
+def validate_url_extra(value):
+    try:
+        # Try normal validator first
+        validate_url(value)
+    except ValidationError:
+        # Fallback to absolute urls
+        if re.match(r'^tel:[0-9\+\#\*\-\.\(\)]+$', value):
+            pass
+        elif re.match(r'^mailto:\S+$', value):
```

Comment:
I'm totally not into regex at all. Does this check if after `mailto:` there follows a valid email address?